src/components/dashboard/LabelContainer/LabelContainer.js

```diff
@@ -0,0 +1,29 @@
+import { MdClear } from 'react-icons/md';
+import Typography from 'components/common/Typography/Typography';
+import React from 'react';
+import './LabelContainer.css';
+
+const LabelContainer = ({ labels, handleLabelDelete }) => {
+  return (
+    <div className="LabelContainer__root">
+      {labels.map((label, idx) => (
```

Comment:
Check for labels when it is null and handle it accordingly

Reply:
It has been already checked in the Parent component. This component only renders when parent component data has label in it.